frappe/integrations/frappe_providers/__init__.py

```diff
@@ -1,13 +1,27 @@
 # imports - standard imports
-import sys
-
 # imports - module imports
+
 from frappe.integrations.frappe_providers.frappecloud import frappecloud_migrator
 
+# def is_working_press(remote_site_name: str):
```

Comment:
Can drop this if its unused

Reply:
umm.. i just kept it thee hoping can be of use someday . hehe.